src/vm/gchandleutilities.h

```diff
@@ -63,95 +63,132 @@ inline BOOL ObjectHandleIsNull(OBJECTHANDLE handle)
 #ifndef DACCESS_COMPILE
 
 // Handle creation convenience functions
+inline OBJECTHANDLE CreateHandleCommon(IGCHandleStore* store, Object* object, int type)
```

Comment:
Nit: I think it would look a bit better to have the OOM check duplicated instead of having yet another layer.